connector-runtime/spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/ZeebeHealthIndicator.java

```diff
@@ -47,6 +51,10 @@ protected void doHealthCheck(Builder builder) {
     if (numBrokers > 0 && anyPartitionHealthy) {
       builder.up().withDetails(details);
     } else {
+      LOG.warn(
+          "Zeebe health check failed: numBrokers={}, anyPartitionHealthy={}",
+          numBrokers,
+          anyPartitionHealthy);
```

Comment:
We probably wont catch it when 
```
var topology = camundaClient.newTopologyRequest().send().join();
```
throws an exception. Can we do this in a try catch where we log and rethrow?